ci/kokoro/docker/e2e/Dockerfile

```diff
@@ -0,0 +1,152 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:18.04
+
+# Dependencies
+RUN \
+  apt-get update && \
+  apt-get -y install \
+          autoconf \
+          build-essential \
+          clang-tidy \
+          cmake \
+          curl \
+          gcc \
+          git-core \
+          golang \
+          g++ \
+          libtool \
+          libssl-dev \
+          make \
+          pkg-config \
+          wget \
+          vim \
+          zip \
+          zlib1g-dev
+
+# Build directory
+RUN mkdir -p /build && \
```

Comment:
AFAIK `WORKDIR /build` would do this for you.

Reply:
Done